src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -547,12 +549,14 @@ function prepareQuestionForm(type) {
         $('#questionTypeHeader').html(FEEDBACK_QUESTION_TYPENAME_TEXT);
         
         hideAllNewQuestionForms();
+        hideInvalidRecipientTypeOptionsForNewlyAddedQuestion();
```

Comment:
Just curious, is it possible for both these hide functions to be put outside the switch-case construct? Right now it's duplicated as much as the different number of question types.